vlib/v/checker/tests/like_operator_outside_orm_error.out

```diff
@@ -1,4 +1,4 @@
-vlib/v/checker/tests/like_operator_outside_orm_error.vv:4:15: error: unexpected name `like`, expecting `,`
+vlib/v/checker/tests/like_operator_outside_orm_error.vv:4:15: error: only `c`, `r`, `js` are recognized string prefixes, but you tried to use `like`
     2 |     name := 'Luke'
     3 | 
     4 |     println(name like 'L%')
```

Comment:
The only test that is updated. Though imho an improvement, the old error informing about an expecting a comma which would be wrong to use here.